src/main/scala/org/kframework/kore/parser/MiniToTextToMini.scala

```diff
@@ -0,0 +1,28 @@
+package org.kframework.kore.parser
```

Comment:
@lucaspena @ehildenb @msaxena2  You may want to drop this file at all. See https://github.com/kframework/kore/issues/16